styleguide.css

```diff
@@ -0,0 +1,273 @@
+/* Normalize */
+
+html {
+	font-family: sans-serif;
+	-webkit-text-size-adjust: 100%;
+	-ms-text-size-adjust: 100%;
+}
+
+body {
+	margin: 20px;
+}
+
+article,
+aside,
+details,
+figcaption,
+figure,
+footer,
+header,
+menu,
+nav,
+section,
+summary {
+	display: block;
+}
+
+/* Vremena Grotesk */
+
+@font-face {
+    font-family: 'vremena_groteskregular';
```

Comment:
A [better strategy](https://stackoverflow.com/a/2436830) is to keep the same `font-family` name for each declaration but deal with CSS weight/style attributes given the font file you load.